README.md

```
@@ -1,142 +1,147 @@
- <a href="#toc" title="Table of Contents"><kbd><b>Table of Contents ⮭</b></kbd></a><br>
- <a href="#app-top"><kbd><b>back to top ⮭</b></kbd></a>
-</div>
+# <a id="app-top"></a> Practice Modern JavaScript [![MIT License](https://img.shields.io/badge/-MIT-56A902.svg?style=flat-square&maxAge=2592000 "MIT License")](LICENSE) [![Tweet Practice Modern JavaScript](http://i.imgur.com/1Y9LBHw.png "Tweet Practice Modern JavaScript")](https://twitter.com/intent/retweet?tweet_id=726917916843601920&related=abhisek%3ACreator,babeljs%3ATranspiler)   [![Open Source Helpers](https://www.codetriage.com/abhisekp/practice-modern-javascript/badges/users.svg)](https://www.codetriage.com/abhisekp/practice-modern-javascript)
+
+> :sunglasses: **Level up your JavaScript skills** — a ready-to-run boilerplate
+> that gets you practicing **Modern JavaScript** (ES2015+) in minutes, with a
+> live-reloading setup, real examples, and zero configuration headaches.
+
+A boilerplate to setup and practice Modern JavaScript
+
+[![](http://i.imgur.com/rzyanJt.gif)](#app-top)  
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)